requirements/edx/base.txt

```diff
@@ -1091,9 +1087,9 @@ xblock==1.5.1
     #   xblock-google-drive
     #   xblock-poll
     #   xblock-utils
-git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.3.5#egg=xblock-drag-and-drop-v2==2.3.5
+xblock-drag-and-drop-v2 @ git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.3.5
```

Comment:
This is a change in the latest version of pip-tools where they started using direct reference instead of egg name.
https://github.com/jazzband/pip-tools/blob/master/CHANGELOG.md#630-2021-09-21

PR where changes were made
https://github.com/jazzband/pip-tools/pull/1455